package.json

```diff
@@ -30,6 +30,9 @@
     "test": "jest && jest-it-up",
     "test:watch": "jest --watch"
   },
+  "dependencies": {
+    "crypto-js": "^4.1.1"
```

Comment:
@legobeat : I am open to a better suggestion for crypto library here, we can go with `@noble/suite`.

We requirement is only this one function, Blockaid's team suggested using JS default `crypto` library but I have doubt if that will work in Metamask react-native app.

Reply:
It looks like this package is only used for `calculateSHA256`, which itself is only used to generate a file checksum.

Perhaps we could make that the responsibility of the `StorageBackend` class instead? That class will already be platform-specific, so it can use whatever cryptography library is already available for each platform.

Reply:
That is good point, validate checksum is done after read and before write to storage and thus can go in storage backend instead.

Let me update the PR.